tcl/target/rp2350.cfg

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 
 # RP2350 is a microcontroller with dual Cortex-M33 cores or dual Hazard3 RISC-V cores.
-# https://www.raspberrypi.com/documentation/microcontrollers/rp2350.html
```

Comment:
The RP2350 "documentation URL" (as opposed to the "product URL") would seem to be https://www.raspberrypi.com/documentation/microcontrollers/silicon.html#rp2350